requirements.txt

```diff
@@ -3,10 +3,10 @@ prance[osv]>=0.11 # use apispec[validation] once we no longer fork apispec
 cfenv==0.5.2
 invoke==0.15.0
 psycopg2==2.7.1
-Flask==0.12.4
-Flask-Cors==3.0.2
-Flask-Script==2.0.5
-Flask-RESTful==0.3.5
+Flask==1.02
```

Comment:
Did you mean 1.0.2?

Reply:
Yes, good catch. Please help make the corresponding change.